### DISCUSSION TITLE
What does "Kaskadierung" mean and how to delete recipes?

### DISCUSSION THREAD

**Discussion by rucksman** [TandoorRecipes/recipes#2217] [General]
I tried to delete a (empty) recipe. So in edit mode I clicked on the button "Löschen" at the bottom of the screen. To my great surprise on the following page it states:
```
Kaskadierung The object you are trying to delete is used by the following objects which will also be deleted.
```
What does that mean, and how can I delete this recipe without deleting the other recipe?

**Comment by vabene1111** [comment]
a recipe consists of many realated objetcts,. if you delete the recipe all related objects (listed on that page) will be deleted as well. If there is nothing you need in the list you can safely delete it (usually no problem).
